smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -52,12 +52,9 @@ class PackageManifestGenerator(val ctx: ProtocolGenerator.GenerationContext) {
                 writer.openBlock("targets: [", "]") {
                     writer.openBlock(".target(", "),") {
                         writer.write("name: \$S,", ctx.settings.moduleName)
-                        writer.openBlock("dependencies: [", "],") {
+                        writer.openBlock("dependencies: [", "]") {
                             dependenciesByTarget.forEach { writeTargetDependency(writer, it) }
                         }
-                        writer.openBlock("resources: [", "]") {
-                            writer.write(".process(\"Resources\")")
-                        }
```

Comment:
Removed the `resources` declaration from the service client, since it doesn't have resources anymore.